src/movies.js

```diff
@@ -1,16 +1,212 @@
 // Iteration 1: All directors? - Get the array of all directors.
+function getAllDirectors(movies) {
+    let allDir = movies.map((elem) => {
+        return elem.director;
+    })
+
+    return allDir;
+}
+
 // _Bonus_: It seems some of the directors had directed multiple movies so they will pop up multiple times in the array of directors. How could you "clean" a bit this array and make it unified (without duplicates)?
+function uniqueDirectors(movies) {
+    let dirs = getAllDirectors(movies);
+    let unique = [];
+
+    dirs.forEach((dir) => {
+        if (unique.indexOf(dir) == -1) {
```

Comment:
indexOf works perfect here but since you don't the index if it exists, then using the method .includes would work best! that one already returns a boolean if the string exists in the array or not.